lib/multi-level.js

```diff
@@ -36,28 +57,26 @@ class MultiLevelCache {
     this.shouldCache = opts.shouldCache || (() => true);
 
     this._pendingRefreshes = {};
-    this._initTask = Promise.all(opts.caches.map(cache => {
-      return cache.init().catch(err => {
-        throw err;
-      });
-    }));
+    this._initTask = Promise.all(opts.caches.map(cache => cache.init()));
   }
 
   /**
    * Attempts a cache get
    *
-   * @param {String}   key                    - The cache key
-   * @param {Object}   opts                   - Options for this particular read
-   * @param {Boolean}  [opts.skipCache=false] - Whether the cache should be bypassed (default false)
-   * @param {String}   [opts.maxAge]          - The duration in milliseconds before a cached item expires
-   * @param {ShouldCache} [opts.shouldCache]  - A function to determine whether or not we should cache the item
-   * @param {UpdateFn} updateFn               - async function that defines how to get a new value
+   * @param {String} key The cache key
+   * @param {Object} [opts={}] Options for this particular read
+   * @param {Boolean} [opts.skipCache=false] Whether the cache should be bypassed (default false)
+   * @param {String} [opts.maxAge] The duration in milliseconds before a cached item expires
+   * @param {Number} [opts.maxStaleness=0] The duration, in milliseconds, in which expired cache items are still served
+   * @param {ShouldCache} [opts.shouldCache] A function to determine whether or not we should cache the item
+   * @param {UpdateFn} updateFn async function that defines how to get a new value
    *
    * @async
    * @returns {Promise<GetResult>} a Promise which resolves to an object containing
    * a `value` property and a `fromCache` boolean indicator.
    */
   async get(key, opts, updateFn) {
+    opts = opts || {};
```

Comment:
Why not put the default on the argument?
```javascript
async get(key, opts = {}, updateFn)
```

Reply:
because it's an awkward API and requires you to do:
```js
.get(myKey, undefined, someFn);
```
since passing a `null` doesn't invoke the param default.